components/mercure.rst

```diff
@@ -0,0 +1,48 @@
+.. index::
+   single: Mercure
+   single: Components; Mercure
+
+The Mercure Component
+=====================
+
+    `Mercure`_ is an open protocol allowing to push data updates to web
```

Comment:
Why the indention?

Please add a versionadded constraint like for the other new components

Reply:
@OskarStark it's a quote.
There is no versionadded directive because it's an external component (currently in version 0.1).